microfragments/src/main/java/org/dmfs/android/microfragments/transitions/BackTransition.java

```diff
@@ -110,7 +110,15 @@ public Timestamp timestamp()
     @Override
     public void prepare(@NonNull Context context, @NonNull FragmentManager fragmentManager, @NonNull MicroFragmentHost host, @NonNull MicroFragment<?> previousStep)
     {
-        mResponseCage.pigeon(fragmentManager.popBackStackImmediate()).send(context);
+        boolean result = fragmentManager.popBackStackImmediate();
```

Comment:
Ok, I am not sure I see these cases and the handling clearly, but it's not important at the moment.
Maybe we could spend 5 minutes on this in Dresden, easier to explain it to me in person.